File: Composite/src/com/jkhines/DesignPatterns/CompositePattern/Waitress.java
package com.jkhines.DesignPatterns.CompositePattern;

public class Waitress {
	private MenuNode menu;
	
	public Waitress(MenuNode menu) {
		this.menu = menu;
	}
	
	public void printMenu() {
			menu.print();
	}
}

File: Iterator/Waitress.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace Iterator
{
    class Waitress
    {
        IMenu menu;

        public Waitress(IMenu menu)
        {
            this.menu = menu;
        }

        public void PrintMenu()
        {
            menu.Print();
        }
    }
}
